### PR TITLE
[feature] Limit consecutive SMS sending #683

### LINKED ISSUE #683: [feature] Limit consecutive SMS sending (frontend UI)
Depends on https://github.com/openwisp/openwisp-radius/issues/481.

This app shall use the information returned by the OpenWISP RADIUS API to block the "Send new SMS button" for the amount of seconds specified in the response.

Above the button we shall add a text saying something like: "A new SMS can be requested in X seconds", X being the value returned by the API and will decrease of 1 each second (like a count down).

### PR DESCRIPTION
**Pull request by pandafy**
Closes #683

**Blockers**
- [x] https://github.com/openwisp/openwisp-utils/pull/353
- [x] https://github.com/openwisp/openwisp-radius/pull/485

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ jobs:
 
       - name: Get openwisp-radius
         run: |
-          curl -L https://github.com/openwisp/openwisp-radius/tarball/master -o openwisp-radius.tar.gz
+          curl -L https://github.com/openwisp/openwisp-radius/tarball/issues/481-limit-sms-tokens -o openwisp-radius.tar.gz
```
Comment (pandafy) [inline .github/workflows/ci.yml:81]: TODO: Update this before merging.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -91,6 +91,7 @@ jobs:
       - name: Installing OpenWISP Radius
         run: |
           cd openwisp-radius && pip install -U -e ".[saml]" "Django~=4.2.0"
+          pip install --force-reinstall --no-deps "https://github.com/openwisp/openwisp-utils/tarball/fallback-positiveintegerfield"
```
Comment (nemesifier) [inline .github/workflows/ci.yml:94]: you can remove this

File: client/components/mobile-phone-verification/mobile-phone-verification.js [hunk 3]
```diff
@@ -156,13 +158,20 @@ export default class MobilePhoneVerification extends React.Component {
         phone_number,
       }),
     })
-      .then(() => {
+      .then((response) => {
         // flag SMS as sent to avoid resending it
         sessionStorage.setItem(self.phoneTokenSentKey, true);
         toast.info(t`TOKEN_SENT`);
+        if (response && response.data && response.data.timeout) {
+          this.setState({resendButtonDisabledTimeout: response.data.timeout});
```
Comment (nemesifier) [inline client/components/mobile-phone-verification/mobile-phone-verification.js:166]: let's rename timeout as indicated in openwisp-radius, I think `cooldown` is easy to understand.

**Review by nemesifier — CHANGES_REQUESTED** [review]

**Review by nemesifier — CHANGES_REQUESTED** [review]
Please make sure the button to resend the SMS is dimmed while disabled, eg: we can lower it's opacity to 0.3/0.4 (make sure the mouseover event doesn't trigger a change of color/opacity).

See also my comment below.

**Review by nemesifier — APPROVED** [review]

**Pull request merged**